### PR TITLE
Openstack Made Easy takeover

### PR DESCRIPTION
**Pull request by caldav**
## Done

* Added OpenStack takeover + form

## QA

Ensure the form works :)

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -6,6 +6,40 @@
 
 {% block outer_content %}
   <!-- Section 1: Hero -->
+<section class="p-strip--image js-takeover is-dark p-takeover--openstack" lang-skip="fr de">
+  <div class="row u-vertically-center">
+    <div class="col-7 u-fade-left--medium">
+    <img src="https://assets.ubuntu.com/v1/35515576-openstack-cloud.svg" width="300px">
```
Comment (anthonydillon) [inline templates/index.html]: Missing `alt` tag

**Comment by anasereijo** [comment]
This will be discussed at Marketing weekly meeting.

**Comment by pmahnke** [comment]
@caldav I think it would be nice to discuss something with this much UX/design impact with the team before building it.  it is a pretty major departure from what we normally do and is worth considering before just putting in a PR.

**Comment by caldav** [comment]
@pmahnke and @anasereijo, apologies for jumping the gun here! I've now turned it into a regular takeover + landing page.

**Pull request merged**